packages/configtest/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import { validate } from 'webpack';
+
+class ConfigTestCommand {
+    async apply(cli): Promise<void> {
+        const { logger } = cli;
+
+        await cli.makeCommand(
+            {
+                name: 'configtest <config-path>',
+                alias: 't',
+                description: 'Test your webpack configuration against validation errors.',
+                usage: '[configs]',
+                pkg: '@webpack-cli/configtest',
+            },
+            [],
+            async (configPath, program) => {
+                if (program.args.length > 1) {
+                    logger.error('Only one configuration can be validated at a time.');
+                    process.exit(2);
+                }
+
+                const { options } = await cli.resolveConfig({ config: [configPath] });
+
+                //eslint-disable-next-line @typescript-eslint/no-explicit-any
+                const validationErrors: any = validate(options);
+
+                if (validationErrors) {
+                    logger.error(validationErrors);
```

Comment:
process.exit(2) also please check how validate messages looks, we don't need stack

Reply:
WIP

Reply:
It's huge, how to reduce it?

![Screenshot at 2021-01-05 17-45-35](https://user-images.githubusercontent.com/46647141/103645458-05665c00-4f7e-11eb-98f5-ac720ea63cb8.png)

Reply:
Logic here https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/webpack-cli.js#L1183